### ISSUE TITLE
MAX V1 - Skybox reflections on enclosed interiors is back

### ISSUE THREAD

**Issue by OldFlak** [TheGameCreators/GameGuruRepo#4588]
So not much time for any dev work atm - fired up MAX for another quick look see at V1
Tried an older level that uses a space skybox and noticed interiors are affected by outside sky - and old bug returned
- this scene has no openings to exterior, the color on the lower part of walls is reflection of skybox

![image](https://user-images.githubusercontent.com/12246318/235299832-47a8f414-6990-4cd2-aa71-ea289442a4e7.png)

**Comment by MonkeyFrogStudio** [comment]
Not that you should have to do this, but do you have double-sided turned on for your materials for these corridors? If not, then perhaps it's because, from the other side, the object is "see-through", thus allowing the skybox colors to influence your interior.

**Comment by OldFlak** [comment]
Yes - All are double-sided. 
As mentioned, it was an old bug that has returned. Interestingly it was all fine up until the V1 release, so something there has broken it again.

**Comment by MonkeyFrogStudio** [comment]
Ah, well. I was just fishing. Thanks for responding. :)

**Comment by LeeBamberTGC** [comment]
@OldFlak yes all level objects will revert to the global env map. You can resolve this by adding the new Environment Probes inside the tunnel to inform the lighting system to take the env map from the interior walls. Works a charm:

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/db9ae874-6d76-4d6c-9e98-f9c4e71b20f7)

**Comment by OldFlak** [comment]
Thanks Lee - is there any performance concerns with the probes for levels where many are required?

**Comment by OldFlak** [comment]
Also it still occurs even with probes - when I added the probes it fixed it. Closed MAX and reopened the level and it still has the issue.

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/12246318/3dfe15c1-bc6f-4654-a33e-df3b6f158905)
The pruple/redish colr is from the skybox :)

**Comment by LeeBamberTGC** [comment]
@OldFlak Can you post a shot of this same scene from the level editor, and the probe you have placed, opened on its properties, thanks!  In theory, the manual prove should 'replace' any global env map that contains that pink color.

**Comment by OldFlak** [comment]
Hi Lee thanks again for looking at this - sorry to keep annoying you with Naeulrec! :)

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/12246318/5adc12a6-a3e0-4b8f-96cf-012528e47272)

**Comment by LeeBamberTGC** [comment]
@OldFlak Can you also send the FPM level. I know the media will not load, but all the probe and sky settings will be there and I can investigate pink a little more, thanks!  P.S. The manual probe setup looks fine.

**Comment by OldFlak** [comment]
Hi Lee - Map sent, I also included the skybox in case it helps :)

**Comment by LeeBamberTGC** [comment]
@OldFlak Thanks, the last files you sent really helped. Turns out that distant env map renderings culled out apparent small objects, but the scaling was way off. The next EXP build now includes even distant objects when making the dynamic env maps so that you no longer get holes through to the sky in your placed environmental probes. I have screenshotted a level that uses the Offworld DLC which I ended up using to test an enclosed shiny corridor scene with a nebula sky :)

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/269f7ec4-4276-4884-ae4f-b2154e52d364)
